src/components/molecules/NavBar/NavBar.tsx

```diff
@@ -144,7 +145,11 @@ const NavBar: React.FC<NavBarPropsType> = ({
     setEventScheduleVisible(!isEventScheduleVisible);
   }, [isEventScheduleVisible]);
   const hideEventSchedule = useCallback((e) => {
-    if (e.target.closest(`.${navBarScheduleClassName}`)) return;
+    if (
+      e.target.closest(`.${navBarScheduleClassName}`) ||
+      e.target.closest(`.${eventModalClassName}`)
+    )
+      return;
```

Comment:
> Why do we need to add the eventModalClassName here as well?

because EventModal renders independently (not as a nested component of the `NavBar` or `Schedule`).

```ts
e.target.closest(`.${navBarScheduleClassName}`)
// false
```
this condition returns `false` as the click was on the backdrop area of the modal. And as the click was made out of Schedule area, the Schedule hides. 

but IMO it's not comfortable to open schedule every time when we want to open EventModal to see the event details.

<hr>

in fact, I guess it'd make more sense if I specified here `.modal` instead of `.EventModal` class name, that would prevent adding other modal class names as an exception in future.

Reply:
`.EventModal` -> `.modal` : https://github.com/sparkletown/sparkle/pull/1426/commits/d5d4dec2dc6d8fc1e6dc643953a459458a1f3110

Reply:
@sunny-viktoryia so, if I understand correctly it's there to prevent closing Schedule, when you want to exit any modal, correct?

Reply:
@mike-lvov correct.

Reply:
I'm okay with it then. @0xdevalias feel free to unresolve, if you have any better approach for this